includes/discovery/sensors/ipmi.inc.php

```diff
@@ -6,17 +6,17 @@
 // IPMI - We can discover this on poll!
 if ($ipmi['host'] = get_dev_attrib($device, 'ipmi_hostname')) {
     echo 'IPMI : ';
-
+    $ipmi['port'] = get_dev_attrib($device, 'ipmi_port');
     $ipmi['user'] = get_dev_attrib($device, 'ipmi_username');
     $ipmi['password'] = get_dev_attrib($device, 'ipmi_password');
     $ipmi['kg_key'] = get_dev_attrib($device, 'ipmi_kg_key');
 
     $cmd = [Config::get('ipmitool', 'ipmitool')];
     if (Config::get('own_hostname') != $device['hostname'] || $ipmi['host'] != 'localhost') {
```

Comment:
This will break of ipmi_port is not set.  is it always set or only sometimes?

Reply:
Yes, you are right, didnt notice. We also found out that the form to set the IPMI parameters seems to be ignoring the default port (623) and it writes 0 if not manually specified

Reply:
We found a solution to that, we took it from includes/polling/pimi.inc.php, I'll update the pull request